Makefile

```diff
@@ -16,25 +19,25 @@ help:
 .PHONY: clean
 clean:			## Removes all created artefacts
 clean:
-	mysql -u root -e "DROP DATABASE IF EXISTS fidry_alice_data_fixtures;"
-	mongo fidry_alice_data_fixtures --eval "db.dropDatabase();"
+	mysql -u root -h 127.0.0.1 --port=3307 -e "DROP DATABASE IF EXISTS fidry_alice_data_fixtures;"
```

Comment:
maybe you can move that `mysql -u root -h 127.0.0.1 --port=3307` into a `MYSQL_BIN` variable as well although it would be nice to have it in a docker container as well